server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/ShardSizeTermsIT.java

```diff
@@ -86,20 +86,24 @@ public void testShardSizeEqualsSizeString() throws Exception {
                 terms("keys").field("key")
                     .size(3)
                     .shardSize(3)
+                    .showTermDocCountError(true)
                     .collectMode(randomFrom(SubAggCollectionMode.values()))
                     .order(BucketOrder.count(false))
             )
             .get();
 
         Terms terms = response.getAggregations().get("keys");
         List<? extends Terms.Bucket> buckets = terms.getBuckets();
-        assertThat(buckets.size(), equalTo(3));
+        assertEquals(3, buckets.size());
```

Comment:
@jed326 why we are changing these asserts? `assertThat` prints significantly better error message

Reply:
This `assertThat` is `org.junit.Assert.assertThat` which is deprecated, so I started changing this in the methods I touched. We could switch to `org.hamcrest.MatcherAssert.assertThat`, which is what junit recommends, or I can just revert this change.

It's probably easier to make that hamcrest change in all places at once in a big batch so I can just revert this instead.

Reply:
> It's probably easier to make that hamcrest change in all places at once in a big batch so I can just revert this instead.

Please, thank you

Reply:
Already done!